lib/rubocop/cop/layout/indent_hash.rb

```diff
@@ -18,8 +18,29 @@ module Layout
       #
       # This default style is called 'special_inside_parentheses'. Alternative
       # styles are 'consistent' and 'align_braces'. Here are examples:
+      # @example
+      #   # bad
+      #     special_inside_parentheses
+      #     hash = {
+      #       key: :value
+      #     }
+      #     and_in_a_method_call({
+      #       no: :difference
+      #                          })
+      #     # consistent
+      #     hash = {
+      #       key: :value
+      #     }
+      #     but_in_a_method_call({
+      #                            its_like: :this
+      #                           })
+      #     # align_braces
+      #     and_now_for_something = {
+      #                               completely: :different
+      #     }
       #
-      #     # special_inside_parentheses
+      #   # good
```

Comment:
It is good to such `# bad` and `# good` comment will be aligned with indented example code. The following address is the template code.
https://github.com/bbatsov/rubocop/blob/v0.51.0/lib/rubocop/cop/generator.rb#L19-L30